cumulusci/core/config/project_config.py

```diff
@@ -240,21 +241,33 @@ def _split_repo_url(self, url):
         git_info = {"url": url, "owner": owner, "name": name}
         return git_info
 
+    def git_path(self, tail=None):
+        """Returns a Path to self.repo_root with
```

Comment:
you mean to the .git folder in repo_root...

Reply:
🤦